src/Panel.js

```diff
@@ -94,15 +94,21 @@ class Panel extends React.Component {
   }
 
   renderAnchor(header, id, role, expanded) {
+    const props = {
+      role,
+      href: id && `#${id}`,
+      onClick: this.handleClickTitle,
+      'aria-controls': id,
+      'aria-expanded': expanded,
+      'aria-selected': expanded,
+    };
+
+    if (!expanded) {
+      props.className = 'collapsed';
```

Comment:
you don't need to do all this, just keep the previous code but do `className={expanded ? null : 'collapsed'}` if you really don't want to use `classNames` there.